internal/service/userService.go

```diff
@@ -0,0 +1,69 @@
+package service
```

Comment:
We should rename this package to something like user_service. It's not clear what the service is when looking at function definitions that reference this package.

Reply:
But the user_service is a service, I'm following MVC design, so service is the correct name to it. Since I don't have more services, I don't need to create a sub-package like user or something.

Reply:
Your correct that you don't need to change the package. The service name should be changed to UserService instead of just Service. For an example of a handler using a service with a meaningful name see
https://github.com/wexinc/ps-tag-payment-scheduler/blob/a6a283834641399513ba31f31bac27fda8aa7279/internal/adapters/httpserver/payment_handler.go#L24-L27

Reply:
I don't get your comment, can you explain better what I need to change?